### PR TITLE
tests:fix bug caused by gc with stale command

### PR DESCRIPTION
**Pull request by AndreMouche**
hi, 
     This PR is used to fix test's bug cased by gc with `stale command`, the related issue is https://github.com/pingcap/tikv/issues/1740

@disksing @hhkbp2 PTAL

### CODE REVIEW

**Comment by disksing** [comment]
LGTM.

File: tests/storage/test_storage.rs [hunk 1]
```diff
@@ -375,21 +374,14 @@ pub fn test_txn_store_gc_multiple_keys_cluster_storage(n: usize, prefix: String)
...
-        }
     }
+
     for k in &keys {
-        store.update_with_key(&mut cluster, k);
-        let store_id = store.ctx.get_peer().get_store_id();
-        if stores.remove(&store_id) {
-            store.gc_ok(30); // clear data which commit_ts < 30
-        }
+        // clear data which commit_ts < 30
```
Comment (hhkbp2) [inline tests/storage/test_storage.rs]: s/which/whose/

**Comment by hhkbp2** [comment]
Rest LGTM

**Comment by hhkbp2** [comment]
LGTM

**Pull request merged**